test/proxysql/proxysql.cnf

```diff
@@ -0,0 +1,141 @@
+datadir="/var/lib/proxysql"
+errorlog="/test/proxysql/proxysql.log"
+
+admin_variables=
+{
+    # https://proxysql.com/documentation/global-variables/admin-variables Note
+    # that while admin variables are documented with an 'admin-' prefix, they
+    # are specified in the configuration with the prefix stripped.
+    mysql_ifaces="0.0.0.0:6032"
+    # admin:admin is only used for local connections. For remote connections,
+    # use radmin:radmin
+    admin_credentials="admin:admin;radmin:radmin"
+    # Web UI is disabled by default.
+    web_enabled="true"
+    web_port=6080
+    # These are the credentials used for the web interface.
+    stats_credentials="stats:stats"
+    debug=true
+}
+
+mysql_variables=
+{
+    threads=4
+    max_connections=10240
+    have_compress=true
+    poll_timeout=2000
+    interfaces="0.0.0.0:6033"
+    stacksize=1048576
+    # Match MariaDB
+    max_allowed_packet=16777216
+    # Allow up to 20 seconds to find a server, to limit how many failures
+    # Boulder sees when we do a primary swap
+    connect_timeout_server=20000
+    connect_timeout_server_max=20000
+    # Monitoring
+    monitor_username="proxysql"
+    monitor_password=""
+    monitor_history=600000
+    monitor_connect_interval=60000
+    monitor_ping_interval=10000
+    monitor_read_only_interval=1000
+    monitor_read_only_timeout=500
+    monitor_writer_is_also_reader=false
+    commands_stats=true
+    sessions_sort=true
+    connect_retries_on_failure=10
+    # Keep 90% of configured connections open
+    free_connections_pct=90
+    connection_warming=true
+    # If mysql_query_rules are marked log=1, they will be logged here. If unset,
+    # no queries are logged.
+    # eventslog_filename="/test/proxysql/events.log"
+    eventslog_filesize=104857600
+    eventslog_default_log=1
+    # The audit logs, if unset, are not logged. If set, every connection gets
+    # logged. Given Boulder's connection strategy, this can be noisy.
+    # auditlog_filename="/test/proxysql/audit.log"
+    auditlog_filesize=104857600
+}
+
+mysql_servers =
+(
+    {
+ 		address="boulder-mysql"
+ 		port=3306
+ 		hostgroup=0
+ 		max_connections=100
+        max_latency_ms=200
+ 	}
```

Comment:
seems like some odd indentation going on here

Reply:
I believe this is virtually identical to the indentation on lines `91-137` in this file and the example configuration in the official repository:

https://github.com/sysown/proxysql/blob/8af813715af7dd927c14a6002e5302d062c054fc/etc/proxysql.cnf#L78-L107

Reply:
No, here the opening and closing curly braces are not in the same vertical column as each other, and `max_latency_ms` is indented differently from every other line. I suspect a tabs vs spaces disagreement where your tab settings are making it invisible but mine are causing the disparity to show up.